zmon_aws_agent/aws.py

```diff
@@ -320,6 +325,78 @@ def get_running_elbs(region, acc):
     return lbs
 
 
+def get_running_elbs_application(region, acc):
+    elb_client = boto3.client('elbv2', region_name=region)
+
+    paginator = elb_client.get_paginator('describe_load_balancers')
+
+    elbs = call_and_retry(
+        lambda: paginator.paginate(PaginationConfig={'MaxItems': 1000}).build_full_result()['LoadBalancers'])
+
+    elb_arns = [e['LoadBalancerArn'] for e in elbs]
+
+    arn_chunks = [elb_arns[i: i + 20] for i in range(0, len(elb_arns), 20)]
+
+    tag_desc_chunks = [call_and_retry(elb_client.describe_tags, ResourceArns=arns) for arns in arn_chunks]
+
+    tags = {d['ResourceArn']: d.get('Tags', []) for tag_desc in tag_desc_chunks for d in tag_desc['TagDescriptions']}
+
+    lbs = []
+
+    for e in elbs:
+        arn = e['LoadBalancerArn']
+        name = e['LoadBalancerName']
+
+        tg_paginator = elb_client.get_paginator('describe_target_groups')
+        target_groups = call_and_retry(
+            lambda: tg_paginator.paginate(LoadBalancerArn=arn).build_full_result()['TargetGroups'])
+
+        lb = {
+            'id': entity_id('elb-{}[{}:{}]'.format(name, acc, region)),
+            'type': 'elb',
+            'infrastructure_account': acc,
+            'region': region,
+            'created_by': 'agent',
+            'elb_type': 'application',
+            'dns_name': e['DNSName'],
+            'host': e['DNSName'],
+            'name': '/'.join(arn.rsplit('/')[-3:]),  # name used by Cloudwatch!
+            'elb_arn': arn,
```

Comment:
maybe just `arn`? so we can reuse the field name for other entities (all AWS resources have ARNs)